src/devices/src/virtio/vsock/unix/muxer.rs

```diff
@@ -1320,4 +1320,23 @@ mod tests {
 
         assert!(!ctx.muxer.has_pending_rx());
     }
+
+    #[test]
+    fn test_regression_handshake() {
```

Comment:
What I noticed at another PR is that they also added a comment with the GitHub issue link that was related to the regression test. It would be nice to have it here as well.

Reply:
Link added